### PR TITLE
setup README: Native on 20.04, WSL on windows, otherwise devcontainer

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

**Comment by haraschax** [comment]
@fredyshox Draft changes for what we talked about, want to go from here? Let's make sure the devcontainers work well on linux and macos before we merge this.

**Comment by fredyshox** [comment]
I guess since there's WSL on windows, we don't need to support it in devcontainers for now, since it would require to write bat init script

**Comment by haraschax** [comment]
Yeah if we get devcontainer on lnix/macos, native on 20.04 and WSL on windows all smooth and reliable, that's already a very good start.

File: tools/README.md [hunk 1]
```diff
@@ -38,7 +36,7 @@ Build openpilot with this command:
 scons -u -j$(nproc)
 ```
 
-### Dev Container
+## Dev Container on any linux or maxOS
```
Comment (royjr) [inline tools/README.md:39]: macOS*

**Comment by adeebshihadeh** [comment]
Let's also add a notice that needs confirmation at the beginning of mac_setup.sh

**Comment by fredyshox** [comment]
Added setup prompt here: https://github.com/commaai/openpilot/pull/29939

**Comment by jnewb1** [comment]
we should also fix this issue if we want to recommend it to non-20.04 linux users: https://github.com/commaai/openpilot/pull/29704

**Pull request merged**